makefile

```diff
@@ -3,8 +3,8 @@ include redis.mk
 
 all: redis
 
-active_record: test_active_record
+active_record: start_pg_primary create_test_database stop_pg test_active_record
```

Comment:
Is this `stop_pg` or `stop_pg_primary`?

Reply:
`stop_pg` which runs `stop_pg_primary` and `stop_pg_replica`.